data/sql/eluna_create/create_mydb.sql

```diff
@@ -0,0 +1,3 @@
+CREATE DATABASE `acore_eluna` DEFAULT CHARACTER SET UTF8MB4 COLLATE utf8mb4_unicode_ci;
+
+GRANT ALL PRIVILEGES ON `acore_eluna` . * TO 'acore'@'localhost' WITH GRANT OPTION;
```

Comment:
```suggestion
GRANT ALL PRIVILEGES ON `acore_eluna` . * TO 'acore'@'localhost' WITH GRANT OPTION;

```

Reply:
Sorry Pango, I'm just dumb or i dont see the difference between previous and after? 
A newline?

![image](https://github.com/user-attachments/assets/1c3a23cf-d39f-4365-8591-5965bfc471c6)

Reply:
Yes, on GitHub if you don't add a line at the end, this symbol appears. But don't apologize for asking or say you're stupid. We all had doubts the first time we made a contribution.

![image](https://github.com/user-attachments/assets/6268b851-f42b-4be9-90e4-194f92469e5d)

Reply:
What does the stop sign, will it cut the code or something?

Reply:
> What does the stop sign, will it cut the code or something?

It doesn't mean anything, it's just asking you to leave an empty line at the end of the document, that is, make a "line break" as indicated.

Reply:
To implement this i just do 
![image](https://github.com/user-attachments/assets/9f79a5a7-8def-48a6-9f82-e57ff7986741) ?

Reply:
> To implement this i just do ![image](https://private-user-images.githubusercontent.com/16946913/369705619-9f79a5a7-8def-48a6-9f82-e57ff7986741.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcwMjU4NjQsIm5iZiI6MTcyNzAyNTU2NCwicGF0aCI6Ii8xNjk0NjkxMy8zNjk3MDU2MTktOWY3OWE1YTctOGRlZi00OGE2LTlmODItZTU3ZmY3OTg2NzQxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTIyVDE3MTkyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE2Yzg4NTA4ZWQ4ODIyNWY0MDc1YWY0OGU1MjAzMzcxMDI4YWNlNGRjZjcwNDhlMmM5ODI3YjNkNWQ3OTU1YzEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.jQmphtSn8nTkBt5f5IiGVnUluk6G6SpU55R9WCMtz5M) ?

You can accept the suggestion and create a commit from here, or create a new commit from the terminal, as you prefer. If you create it from here, you will have to do a pull afterwards, to continue creating new commits, otherwise you will have problems between your remote history and your local history.